package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ebury/chameleon-components",
-  "version": "0.1.100",
+  "version": "0.1.101",
```

Comment:
Recommendation:
I usually bump the version after get the approvals and just before merge my PR with the last version on master, if you bump the version right now can happen that other people merge another PR update the version and you will have conflicts with the versions

Reply:
Thanks for the advice, I will follow your suggestion